docs/src/pages/components/autocomplete/LimitTags.js

```diff
@@ -1,23 +1,19 @@
 /* eslint-disable @typescript-eslint/no-use-before-define */
 import * as React from 'react';
 import Autocomplete from '@material-ui/core/Autocomplete';
-import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
 import TextField from '@material-ui/core/TextField';
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    width: 500,
-    '& > * + *': {
-      marginTop: theme.spacing(3),
-    },
-  },
-}));
-
 export default function LimitTags() {
-  const classes = useStyles();
-
   return (
-    <div className={classes.root}>
+    <Box
+      sx={{
+        width: 500,
+        '& > * + *': {
```

Comment:
This could go in a line.  Why don't we use `:not(style)` here?

Reply:
Best to remove the whole Box. There is a single demo: https://next.material-ui.com/components/autocomplete/#limit-tags

Reply:
Didn't noticed, will remove it.